Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -204,15 +204,8 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
         if indexPath.item < 6 {
             let cell = collectionView.dequeueReusableCellWithReuseIdentifier(ThumbnailIdentifier, forIndexPath: indexPath) as ThumbnailCell
             cell.textLabel.text = site.title.isEmpty ? site.url : site.title
-            cell.image = nil
-
-            if let url = NSURL(string: site.url) {
-                profile.thumbnails.get(url) { (thumbnail: Thumbnail?) in
-                    if let thumbnail = thumbnail {
-                        cell.image = thumbnail.image
-                    }
-                }
-            }
+            let key = SDWebThumbnails.getKey(site.url)
+            cell.imageView.sd_setImageWithURL(NSURL(string: key))
```

Comment:
Also, this commit breaks the placeholder thumbnail image (test case: thebnich.com, which stays blank since it redirects). It worked before because we have a custom `image` setter on `ThumbnailCell`, so I guess we'll need to figure out something else here. I noticed `sd_setImageWithURL` accepts a `placeholderImage`, so that might be useful here. But the `image` setter also used a different `alignment` and `contentMode` for the placeholder image...I'm not sure how we'll handle that now.